content/faq/faq.mdx

```diff
@@ -145,3 +145,40 @@ Now default values within the schema file can be used:
   "default": "Max"
 }
 ```
+## How can I use multipleOf with values smaller 1?
+
+JSON Forms uses AJV for the validation of data. By default, the `multipleOf` option of AJV only works with integers. If you are using fractional dividers, you also need to set the `multipleOfPrecision` option in AJV and pass your AJV instance to JSON Forms. To create the AJV instance with suitable options, you can call the `createAjv(options)` method exported by JSON Forms Core.
```

Comment:
Please use a new line for each sentence. This makes it easier to diff changes in the future :)